sys/clif/clif.c

```diff
@@ -323,7 +327,21 @@ ssize_t clif_get_attr(const char *input, size_t input_len, clif_attr_t *attr)
         attr->key_len = pos - attr->key;
     }
 
+    /* either the value is unquoted (0) or quoted (2) */
+    if (quotes != 0 && quotes != 2) {
```

Comment:
The previous suggestion would allow you to only check for `quotes != 0` here.

Reply:
How to detect `rt=""` if quotes are decremented?

Reply:
When the `if (scan_value && !attr->value_len) {` is moved above this check (see my proposal below) it will be detected.

Reply:
See 5d677575cf76c32f39fca04b4e30749d05f4160a